test/controllers/ApplicationControllerSpec.scala

```diff
@@ -117,16 +110,6 @@ class ApplicationControllerSpec extends PlaySpec with GuiceOneServerPerSuite wit
         status(result) must be(SEE_OTHER)
       }
 
-      "be redirected to the feedback page for ATED service" in new Setup {
```

Comment:
This test should have remained in place as its testing logout service and not feedback - may be badly worded but thats the controller its calling

Reply:
Added back in